packages/indexer-client/src/types/clientTypes.ts

```diff
@@ -183,11 +195,25 @@ export interface GetIndexerProductSnapshotsParams {
   limit: number;
 }
 
-export type GetIndexerProductSnapshotsResponse = (Market & {
-  timestamp: BigDecimal;
+export interface IndexerProductSnapshot extends Market {
   submissionIndex: string;
+}
+
+export type GetIndexerProductSnapshotsResponse = (IndexerProductSnapshot & {
```

Comment:
re: timestamp -- the BE currently doesn't return it as it's part of the txs, if we need it would have to make some backend changes.

> Also, thoughts on using composition instead of inheritance? (nesting market as a property and not extending Market)

so instead of 
```
export interface IndexerProductSnapshot extends Market {
  submissionIndex: string;
}
```
do something like
```
export interface IndexerProductSnapshot {
  product: Market
  submissionIndex: string;
}
```
?

I think I prefer inheritance in this case since it's just a few extra fields

Reply:
> Timestamp - fine to omit, should we remove it as part of the single-product intf as well?

ye, we can remove it -- after all it's only at the sdk level anyways and if ppl want it they can just fetch it from the txs (will also help clean up the types, etc) will fix!

Reply:
fixed!